polytope/polytope.py

```diff
@@ -570,6 +564,18 @@ def _rotate(polyreg, i=None, j=None, u=None, v=None, theta=None, R=None):
     return R
 
 
+def _Givens_rotation_matrix(i, j, theta, N):
```

Comment:
I am aware that "Givens" is a person's name, but by PEP8 function names need to be lowercase.

Most of the module `polytope.polytope` is algorithms implemented as functions, so the contents are along the same lines. Whether it is better or not to make these functions public I am not sure. Maybe we can wait to see how frequently users need to call them directly.

In general, one problem that will arise from splitting the `polytope` module is cyclic imports. This may not be an issue for rotation functions, but it is for other functions. What would be the motivation for placing the functions in another file?